docs/graphql/core/deployment/deployment-guides/heroku.rst

```diff
@@ -65,20 +65,20 @@ Step 2: Create an app with **--stack=container**
 ************************************************
 
 Use the `Heroku CLI <https://devcenter.heroku.com/articles/heroku-cli>`__ to create a new Heroku app from inside the cloned directory. Let's call
-the app ``graphql-on-postgres``.
+the app ``todo-hasura1``.
```

Comment:
Why did we change the name here? In the screenshot above it's `graphql-on-postgres`. We should be consistent.

Reply:
@codeliezel this is not resolved yet. We either have to change the screenshot to reflect the new name (``todo-hasura1``) or change the name back to ``graphql-on-postgres``. Personally, I think the previous name was fine.